src/web/components/App.tsx

```diff
@@ -330,6 +336,12 @@ export const App = ({ CAPI, NAV }: Props) => {
 		});
 	}, []);
 
+	useOnce(() => {
+		setBrazeMessages(
+			buildBrazeMessages(isSignedIn as boolean, CAPI.config.idApiUrl),
```

Comment:
if you set the default `isSignedIn` to `false` in the useState you shouldnt need to case it here

Reply:
Ah right yeah that's true. This is needed because it starts as `undefined`. I'm not sure about changing the default from `undefined` to `false` as other things may be relying on the current behaviour. Perhaps worth revisiting in another PR?

Reply:
I think there's a specific use case for it being undefined

`undefined` = we're checking 🤷 
`false` = definately not signed in
`true` = signed in

It's probably worth adding a comment about this